core/certs.go

```diff
@@ -89,7 +88,7 @@ func Generate(certPath string, keyPath string, host string) error {
 		os.Exit(1)
 	}
 	if err != nil {
-		log.Printf("failed to generate private key: %s", err)
+		fmt.Printf("failed to generate private key: %s", err)
```

Comment:
def want to use `log` not `fmt`. when this moves out of core (ssl module?) you'll create a `log` var for that entire package... (see `cmd`)

Reply:
cool, thanks, will do